.github/ISSUE_TEMPLATE/docs.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Documentation
+about: Use this template for making changes in the docs.
+labels: docs
+---
+
+# Documentation Is
+
+<!-- Please place an x (no spaces!) in all [ ] that apply -->
+
+- [ ] Missing
+- [ ] Needed
```

Comment:
Missing and Needed would seem to be the same thing. I think we need one for Incorrect.

```suggestion
- [ ] Missing/Needed
- [ ] Incorrect
```